test/core/templaters/jinja_test.py

```diff
@@ -943,19 +983,22 @@ def test__templater_jinja_slice_file(raw_file, override_context, result, caplog)
 
     templated_file = make_template(raw_file).render()
     with caplog.at_level(logging.DEBUG, logger="sqlfluff.templater"):
-        _, resp, _ = templater.slice_file(
+        raw_sliced, sliced_file, templated_str = templater.slice_file(
             raw_file, templated_file, make_template=make_template
         )
+    # Create a TemplatedFile from the results. This runs some useful sanity
+    # checks.
+    _ = TemplatedFile(raw_file, "<<DUMMY>>", templated_str, sliced_file, raw_sliced)
```

Comment:
This added line runs the same sanity check we use in production and will help avoid introducing any future "bad" test results.